14.0 mig auth saml create user

matteoopenf opened the pull request.

Commit:
[ADD] Auth SAML create users

Commit:
[FIX] Create user only when it doesn't exist

[FIX] Flake8 & pylint

[FIX] manifest

[FIX] manifest

[UPD] auth_saml_create_user/models/auth_saml.py

Co-Authored-By: Rim BEN DHAOU <bendhaou.rim@gmail.com>

Apply suggestions from code review

[ADD] Tests

[ADD] Tests

[ADD] Starting random password

[FIX Imports and pep8]

[UPD] Tests

[FIX] empty password

[FIX] exception not needed

[FIX] password not needed in write

[UPD] Create saml pass string

[ADD] test secret problem

[FIX] secret null

[ADD]tests: teradown method

[FIX] not need teardown

[FIX] manifest

[FIX] License

[FIX] Maintainers and development_status

Update __manifest__.py

oca-travis committed:
[UPD] Update auth_saml_create_user.pot

OCA-git-bot committed:
[ADD] icon.png

sbidoul committed:
[FIX] lint issues

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
auth_saml_create_user 11.0.1.0.1

OCA-git-bot committed:
# Questa è una combinazione di 20 commit.
# Questo è il primo messaggio di commit:

auth_saml_create_user 11.0.1.0.1

# Questo è il messaggio di commit numero 2:

[UPD] README.rst

# Questo è il messaggio di commit numero 3:

[UPD] README.rst

# Questo è il messaggio di commit numero 4:

[IMP] auth_saml_create_user: black, isort, prettier

# Questo è il messaggio di commit numero 5:

[IMP][auth_saml_create_user] bump version

[IMP][auth_saml_create_user] fix pre-commit

[IMP][auth_saml_create_user] fix ir access model csv

[IMP][auth_saml_create_user] fix ir access model csv

[IMP][auth_saml_create_user] fix tab spaces

# Questo è il messaggio di commit numero 6:

[FIX][auth_saml_create_user] fix version module and classification module

# Questo è il messaggio di commit numero 7:

[FIX][auth_saml_create_user] change status developement

# Questo è il messaggio di commit numero 8:

[FIX][auth_saml_create_user] remove api.multi

# Questo è il messaggio di commit numero 9:

[FIX][auth_saml_create_user] fix pre commit

# Questo è il messaggio di commit numero 10:

[FIX][auth_saml_create_user] monkey patch

# Questo è il messaggio di commit numero 11:

[FIX][auth_saml_create_user] patch test

# Questo è il messaggio di commit numero 12:

[FIX][auth_saml_create_user] patch test

# Questo è il messaggio di commit numero 13:

[FIX][auth_saml_create_user] patch test

# Questo è il messaggio di commit numero 14:

[FIX][auth_saml_create_user] patch test

# Questo è il messaggio di commit numero 15:

[FIX][auth_saml_create_user] more fix

# Questo è il messaggio di commit numero 16:

[FIXC][auth_saml_create_user] fix create

# Questo è il messaggio di commit numero 17:

[FIXC][auth_saml_create_user] fix sign method

# Questo è il messaggio di commit numero 18:

[FIX][auth_saml_create_user] fix model to search

# Questo è il messaggio di commit numero 19:

[FIXC][auth_saml_create_user] fix pre commit

# Questo è il messaggio di commit numero 20:

[FIXC][auth_saml_create_user] fix create

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

matteoopenf committed:
[IMP] auth_saml_create_user: black, isort, prettier

matteoopenf committed:
[IMP][auth_saml_create_user] bump version

[IMP][auth_saml_create_user] fix pre-commit

[IMP][auth_saml_create_user] fix ir access model csv

[IMP][auth_saml_create_user] fix ir access model csv

[IMP][auth_saml_create_user] fix tab spaces

matteoopenf commented:
Hi @pedrobaeza, I see pre-commit fail in other module instead of the module I migrate
I have that:
************* Module auth_user_case_insensitive.tests.test_res_users
auth_user_case_insensitive/tests/test_res_users.py:51: [E8102(invalid-commit), TestResUsers.test_login_login_is_lowercased] Use of cr.commit() directly - More info https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#never-commit-the-transaction

have you any idea to fix that? or I should open an Issue?

matteoopenf commented:
@dave@laslabs.com I see give errore is migrate from your company, can you help me?

matteoopenf commented:
@theangryangel have any idea why fail pre commit?

lorenzoallegrucci commented on auth_saml_create_user/__manifest__.py:
```diff
@@ -0,0 +1,22 @@
+    "website": "https://github.com/OCA/server-auth",
+    "license": "AGPL-3",
+    "category": "Tools",
+    "version": "14.0.1.0.1",
```
non deve ripartire da `14.0.1.0.0` ?

matteoopenf replied:
https://github.com/OCA/server-auth/pull/517/commits/eb0bdfeb51836e539b3b35039f5d7947cc7614da fix

theangryangel commented:
Looking at the actions manifestoo is complaining about the development_status mis-match.

```
auth_saml_create_user (Production/Stable) depends on auth_saml (Beta)
Error: Process completed with exit code 1.
```

Either `auth_saml` needs to be re-classified as `Production/Stable`, or auth_saml_create_user needs to be downgraded to `Beta` to make manifestoo happy :)

I'd suggest that auth_saml_create_user is downgraded personally, but thats just me

matteoopenf commented:
> Beta

thank you for suggestion

matteoopenf committed:
[FIX][auth_saml_create_user] fix version module and classification module

[FIX][auth_saml_create_user] change status developement

[FIX][auth_saml_create_user] remove api.multi

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] monkey patch

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] more fix

[FIXC][auth_saml_create_user] fix create

[FIXC][auth_saml_create_user] fix sign method

[FIX][auth_saml_create_user] fix model to search

[FIXC][auth_saml_create_user] fix pre commit

[FIXC][auth_saml_create_user] fix create

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] remove comment

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix creation user

matteoopenf committed:
[FIX][auth_saml_create_user] fix version module and classification module

matteoopenf committed:
[FIX][auth_saml_create_user] change status developement

matteoopenf committed:
[FIX][auth_saml_create_user] remove api.multi

matteoopenf committed:
[FIX][auth_saml_create_user] fix pre commit

matteoopenf committed:
[FIX][auth_saml_create_user] monkey patch

matteoopenf committed:
[FIX][auth_saml_create_user] patch test

matteoopenf committed:
[FIX][auth_saml_create_user] patch test

matteoopenf committed:
[FIX][auth_saml_create_user] patch test

matteoopenf committed:
[FIX][auth_saml_create_user] patch test

matteoopenf committed:
[FIX][auth_saml_create_user] more fix

matteoopenf committed:
[FIXC][auth_saml_create_user] fix create

matteoopenf committed:
[FIXC][auth_saml_create_user] fix sign method

matteoopenf committed:
[FIX][auth_saml_create_user] fix model to search

matteoopenf committed:
[FIXC][auth_saml_create_user] fix pre commit

matteoopenf committed:
[FIXC][auth_saml_create_user] fix create

matteoopenf committed:
Merge branch '14.0-mig-auth_saml_create_user' of https://github.com/matteoopenf/server-auth into 14.0-mig-auth_saml_create_user

matteoopenf committed:
[FIX][auth_saml_create_user] fix creation user

matteoopenf committed:
Merge branch '14.0-mig-auth_saml_create_user' of https://github.com/matteoopenf/server-auth into 14.0-mig-auth_saml_create_user

matteoopenf committed:
[FIX][auth_saml_create_user] fix creation user

matteoopenf committed:
Merge branch '14.0-mig-auth_saml_create_user' of https://github.com/matteoopenf/server-auth into 14.0-mig-auth_saml_create_user

matteoopenf committed:
[FIX][auth_saml_create_user] fix creation user

matteoopenf committed:
[FIX][auth_saml_create_user] fix creation user

matteoopenf committed:
Merge branch '14.0-mig-auth_saml_create_user' of https://github.com/matteoopenf/server-auth into 14.0-mig-auth_saml_create_user

matteoopenf committed:
[FIX][auth_saml_create_user] more fix

matteoopenf committed:
auth_saml_create_user 11.0.1.0.1

[UPD] README.rst

[UPD] README.rst

[IMP] auth_saml_create_user: black, isort, prettier

[IMP][auth_saml_create_user] bump version

[IMP][auth_saml_create_user] fix pre-commit

[IMP][auth_saml_create_user] fix ir access model csv

[IMP][auth_saml_create_user] fix ir access model csv

[IMP][auth_saml_create_user] fix tab spaces

[FIX][auth_saml_create_user] fix version module and classification module

[FIX][auth_saml_create_user] change status developement

[FIX][auth_saml_create_user] remove api.multi

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] monkey patch

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] more fix

[FIXC][auth_saml_create_user] fix create

[FIXC][auth_saml_create_user] fix sign method

[FIX][auth_saml_create_user] fix model to search

[FIXC][auth_saml_create_user] fix pre commit

[FIXC][auth_saml_create_user] fix create

[FIX][auth_saml_create_user] fix version module and classification module

[FIX][auth_saml_create_user] change status developement

[FIX][auth_saml_create_user] remove api.multi

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] monkey patch

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] patch test

[FIX][auth_saml_create_user] more fix

[FIXC][auth_saml_create_user] fix create

[FIXC][auth_saml_create_user] fix sign method

[FIX][auth_saml_create_user] fix model to search

[FIXC][auth_saml_create_user] fix pre commit

[FIXC][auth_saml_create_user] fix create

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix pre commit

[FIX][auth_saml_create_user] remove comment

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] fix creation user

[FIX][auth_saml_create_user] more fix

[FIX][auth_saml_create_user] fix creation user

matteoopenf committed:
[MIG][auth_saml_create_user] fix conflict

matteoopenf committed:
[FIX][ auth_saml_create_user] fix pre-commit

matteoopenf committed:
[IMP][auth_saml_create_user] add return for method create user

matteoopenf committed:
[FIX][auth_saml_create_user] fix pre commit
